examples/language/llama2/scripts/finetune_7B/finetune.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+################
+#Load your environments and modules here
+################
+
+HOSTFILE=$(realpath hosts.txt)
+
+cd ../..
```

Comment:
Please describe the usage of finetune.sh in README, user might stay in the root directory of llama example.